### ISSUE TITLE
Custom Loss

### ISSUE THREAD

**Issue by Soren-Kierkegaard** [Rincewind1989/AERes#2]
Hello,
I would like to know if is it possible to pass a custum loss to the model ?
I guess the basic Loss is a Ridge Regression here, no.

**Comment by fekoester** [comment]
Hi,

At the moment it is not easily possible to pass your own loss.
But you can change it easily in the code if you want to.
In the attention.py file at line 134 is: "self.mse_loss = nn.MSELoss()", which is the MSE loss from the torch nn library.
You can change this line if you want another loss there.

Or in the function MSELoss in 137 this self.mse_loss function is called and you can put in your own function also there.
Hope that helps.

**Comment by Soren-Kierkegaard** [comment]
Ok, I will check that :!
